src/Servers/Kestrel/test/Libuv.FunctionalTests/Libuv.FunctionalTests.csproj

```diff
@@ -21,6 +21,7 @@
     <Reference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" />
     <Reference Include="Microsoft.AspNetCore.Server.Kestrel" />
     <Reference Include="Microsoft.Internal.AspNetCore.H2Spec.All" />
+    <Reference Include="Microsoft.AspNetCore.Server.Kestrel.Core" />
```

Comment:
Likely don't need this either since the tests already exist elsewhere.

Reply:
It doesn't hurt to keep them either. It ensures these tests use ProjectRefs. When its transitive, it might end up coming from a package.